### ISSUE TITLE
Unable to send email using nylas on self-hosted sync engine

### ISSUE THREAD

**Issue by Jatin21** [nylas/N1#2440]
##### Sync-engine log (if this might help):
`{"server_error": "Network is unreachable", "endpoint": "namespace_api.draft_send_api", "account_id": 1, "level": "error", "timestamp": "2016-06-15T03:48:41.597708Z", "exception": "Traceback (most recent call last):\n\t  File \"/home/ubuntu/nylas/sync-engine/inbox/sendmail/smtp/postel.py\", line 151, in _connect\n    self.connection.connect(host, port)\n\t  File \"/usr/lib/python2.7/smtplib.py\", line 316, in connect\n    self.sock = self._get_socket(host, port, self.timeout)\n\t  File \"/usr/lib/python2.7/smtplib.py\", line 793, in _get_socket\n    new_socket = socket.create_connection((host, port), timeout)\n\t  File \"/usr/local/lib/python2.7/dist-packages/gevent/socket.py\", line 591, in create_connection\n    raise err\n\terror: [Errno 101] Network is unreachable\n", "module": "inbox.sendmail.smtp.postel:155", "greenlet_id": 140705275031216, "event": "SMTP connection error"}`

AND 

` {"server_error": null, "endpoint": "namespace_api.draft_send_api", "account_id": 1, "level": "error", "timestamp": "2016-06-15T03:56:53.469820Z", "exception": "Traceback (most recent call last):\n\t  File \"/home/ubuntu/nylas/sync-engine/inbox/sendmail/smtp/postel.py\", line 151, in _connect\n    self.connection.connect(host, port)\n\t  File \"/usr/lib/python2.7/smtplib.py\", line 316, in connect\n    self.sock = self._get_socket(host, port, self.timeout)\n\t  File \"/usr/lib/python2.7/smtplib.py\", line 793, in _get_socket\n    new_socket = socket.create_connection((host, port), timeout)\n\t  File \"/usr/local/lib/python2.7/dist-packages/gevent/socket.py\", line 591, in create_connection\n    raise err\n\ttimeout: timed out\n", "module": "inbox.sendmail.smtp.postel:155", "greenlet_id": 140705275031216, "event": "SMTP connection error"}`

##### Are there any related issues?

Haven't found any.

##### What operating system are you using?
Ubuntu 14.04.02 LTS

##### What version of N1 are you using?
Version 0.4.40-85cf726

--

**Bug?**
##### Do you have any third-party plugins installed?
No, only the plugins installed by default.

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Gmail, using custom domain name.

##### Is the issue reproducible with a particular attachment, message, signature, etc?

All messages

--

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
...

**Comment by grinich** [comment]
Are you online?

`"server_error": "Network is unreachable"` means it can't connect to your smtp server.

**Comment by Jatin21** [comment]
Yes, But other email client like thunderbird and evolution are able to send emails. Also, my smtp server is gmail smtp server at port 465.

**Comment by deadlocks** [comment]
I am also facing same issue.
